locale/sv_SE/locale.xml

```diff
@@ -1026,135 +938,6 @@ Validering av URL:er kan misslyckas på sökningar som kräver inskickad data.]]
 	 * DEFAULT - Default settings for setup items.
 	 *
 	-->
-	<message key="default.journalSettings.checklist.notPreviouslyPublished">Manuskriptet har inte publicerats tidigare, och ligger inte för publicering i annan tidskrift (eller se bifogad förklaring under Kommentarer under Redaktör).</message>
```

Comment:
I think this is what happened: They are in the wrong file. The translator plugin identifies them as obsolete (does not check if a key has been moved/needs to be moved).

I have started reviewing everything (already hate it) and will repair the few keys that need to stay (will move them to the correct file, from locale.xml to default.xml), update this PR, and will then let you know. Thanks for spotting this, @asmecher .